core/domain/stats_jobs_one_off.py

```diff
@@ -82,6 +87,36 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Implements the map function. Must be declared @staticmethod.
+
+        Args:
+            item: ExplorationAnnotationsModel or
+                StateCounterModel.
+
+        Yields:
+            tuple. 2-tuple in the form ('exploration_id', value).
+                `exploration_id` corresponds to the id of the exploration.
+                `value`: value yielded is:
+                    {
+                        'version': Version of the exploration,
```

Comment:
There should be types here. E.g.

    'starts': int. Number of times exploration was started.